scripts/fix-tslint.ts

```diff
@@ -2,44 +2,46 @@
 
 /// <reference types="node" />
 
-import * as fs from "fs";
-import * as path from "path";
-import JSON = require("comment-json");
+import * as fs from 'node:fs';
+import JSON from 'comment-json';
 
-const home = path.join(__dirname, "..", "types");
+const home = new URL('../types/', import.meta.url);
 
 for (const dirName of fs.readdirSync(home)) {
-    if (dirName.startsWith(".") || dirName === "node_modules" || dirName === "scripts") {
+    if (dirName.startsWith('.') || dirName === 'node_modules' || dirName === 'scripts') {
```

Comment:
formatting changes should be in a separate PR, preferably one that sets up prettier to enforce future formatting.